src/poliastro/twobody/perturbations.py

```diff
@@ -101,6 +102,7 @@ def third_body(t0, state, k, k_third, third_body):
     return k_third * delta_r / norm(delta_r) ** 3 - k_third * body_r / norm(body_r) ** 3
 
 
+@jit
```

Comment:
Just as a clarification, `norm` is not jit-eable because it's meant to preserve units of quantities.